backend/shop/views.py

```diff
@@ -7,20 +7,20 @@
 from django.core.paginator import Paginator
 from django.db import IntegrityError, connection
 from django.shortcuts import render
-from rest_framework import status
+from rest_framework import status, viewsets
 from rest_framework.decorators import api_view, permission_classes
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import IsAuthenticated, IsAuthenticatedOrReadOnly
```

Comment:
Unused import. Please fix your CI so that if will if you have unused imports in your PY files.